src/utils/importFuncGenerator.ts

```diff
@@ -64,57 +77,45 @@ export function createImport(
     );
   const createStructImport = () => createCairoImportStructDefinition(name, path, ast, sourceUnit);
 
-  const warplibFunc = warplibImportInfo.get(path)?.get(name);
+  const warplibFunc = warplibImportInfo.get(path.join('.'))?.get(name);
```

Comment:
If it doesn't break anything we can fix that in another PR. Let me know what you think @rodrigo-pino @cicr99

Reply:
Yes I think all that code will get reduced considerably once we no longer need to track implicits anymore. So I think it's best to wait to update the way Warp treats implicits

Reply:
I see, then let's leave it for later